### ISSUE TITLE
Map Information tool

### ISSUE THREAD

**Issue by dl3sdo** [OpenOrienteering/mapper#2018]
A Happy New Year to all of you.

During the past week I have written a 'Map Information' tool:
![MapInformationMenu](https://user-images.githubusercontent.com/14225666/148091727-dbf21eb1-d30c-4494-997c-04eda959cc29.JPG)
![MapInformation](https://user-images.githubusercontent.com/14225666/148091752-0298eec4-28f8-40df-962f-2565ab0c2be2.JPG)

@dg0yt, @lpechacek: please have a look at it and give me some advice how to proceed (e.g., std::map vs. std::vector and 2 or 3 column layout).
To be discussed: 'Map' item offers the same information as the 'Objects' item.
[Commit](https://github.com/dl3sdo/mapper/commit/724eb20e6441ba27b5e07b2201939c5a0e447b7d)

**Comment by ollesmaps** [comment]
Could that contain also scale info? In Adroid app this info is missing.

**Comment by dl3sdo** [comment]
Yes, of course. Do you want something like this:
![MapScale](https://user-images.githubusercontent.com/14225666/148100594-89e40217-01fe-4932-af68-f1562117ba88.JPG)
Currently the Map Information is only available for Mapper's desktop version.

**Comment by mlerjen** [comment]
What is the use case you have in mind?

**Comment by dl3sdo** [comment]
Basically it's just for information. Maybe also 'strange' objects (as mentioned in #2012) are shown and identified there.
Please keep in mind that this is just the first version of the Map Information.
Any proposal for additional pieces of information is welcome.

**Comment by ollesmaps** [comment]
> Yes, of course. Do you want something like this: ![MapScale](https://user-images.githubusercontent.com/14225666/148100594-89e40217-01fe-4932-af68-f1562117ba88.JPG) Currently the Map Information is only available for Mapper's desktop version.

Perfect! 
For reference I attach issue #1040.

**Comment by dl3sdo** [comment]
BTW: this is the variant with 2 columns:
![MapInformation_2col](https://user-images.githubusercontent.com/14225666/148222439-b8939112-5c3c-4fcc-a255-8d785de57554.JPG)

I slightly prefer the 3 column layout, but would like to get some feedback from others.

**Comment by dl3sdo** [comment]
I have completely rewritten the code: https://github.com/dl3sdo/mapper/commit/97a055bb5d3cf979e6dc7cdb4505e75788363310

**Comment by dl3sdo** [comment]
I also added an overview about the fonts and their usage.
The 'Objects' section now shows for every symbol which colors are used by it.
The 'Colors' section shows all colors and by which symbol every color is used:
![FontsColorsSymbols](https://user-images.githubusercontent.com/14225666/150214637-b01561a0-6220-48c9-9809-89fb90f32270.JPG)

**Comment by dg0yt** [comment]
Fonts:
In a cross-platform scenario, I would be interested in the substitutions chosen for missing fonts. (But this could be added later.)

**Comment by dl3sdo** [comment]
> Fonts: In a cross-platform scenario, I would be interested in the substitutions chosen for missing fonts. (But this could be added later.)

I tried and used
```
auto font_info = QFontInfo(text_symbol->getQFont());
auto font_family_substituted = font_info.family();

```
However, I'm not sure how to show it:
![Fonts1](https://user-images.githubusercontent.com/14225666/150837843-2870c966-9975-4b37-857b-c00bd75eb7e3.JPG)
or:
![Fonts2](https://user-images.githubusercontent.com/14225666/150837917-2884fb03-4020-4212-9640-abd4309a1549.JPG)
or:
![Fonts3](https://user-images.githubusercontent.com/14225666/150838717-24827cdc-74a3-4701-a79d-b1200af250c1.JPG)

**Comment by dl3sdo** [comment]
Pull request: #2125

**Comment by dl3sdo** [comment]
Closed by #2125
